compiler/generator_rust.c

```diff
@@ -1188,14 +1188,23 @@ static void generate(struct generator * g, struct node * p) {
 }
 
 static void generate_start_comment(struct generator * g) {
-
```

Comment:
Please try to avoid reformatting the code copied from other generator files like this.  Martin's code formatting is quite unusual in places, but there's a lot of code shared between the files which would be much better factored out, and keeping the source code matching where it's really the same will make it significantly easier to do that refactoring in the future.